pkg/controller/hvpa/hvpa_controller.go

```diff
@@ -613,14 +624,25 @@ func (r *ReconcileHvpa) reconcileHpa(hvpa *autoscalingv1alpha1.Hvpa) (*autoscali
 		return nil, err
 	}
 
+	updatePolicy := hvpa.Spec.Hpa.UpdatePolicy.UpdateMode
+
 	foundHpa := &autoscaling.HorizontalPodAutoscaler{}
 	err := r.Get(context.TODO(), types.NamespacedName{Name: hpa.Name, Namespace: hpa.Namespace}, foundHpa)
```

Comment:
What about `GenerateName` for HPA and VPA?

Reply:
Are there any blockers for using `GenerateName`? We should pick this in this PR to avoid migration later.

Reply:
Tracked in #22.